cmake/CoverallsGenerateGcov.cmake

```diff
@@ -251,7 +251,18 @@ foreach (GCOV_FILE ${ALL_GCOV_FILES})
 	# ->
 	# /path/to/project/root/subdir/the_file.c
 	get_source_path_from_gcov_filename(GCOV_SRC_PATH ${GCOV_FILE})
-	file(RELATIVE_PATH GCOV_SRC_REL_PATH "${PROJECT_ROOT}" "${GCOV_SRC_PATH}")
+#    message("========== original file path: ${GCOV_SRC_PATH}")
+    if(${GCOV_SRC_PATH} MATCHES "^\\/.*")
+        set(GCOV_SRC_ABS_PATH "${GCOV_SRC_PATH}")
+    else()
+        if(${GCOV_SRC_PATH} MATCHES "\\^.*")
+            string(REGEX REPLACE "\\^" ".." GCOV_SRC_PATH "${GCOV_SRC_PATH}")
```

Comment:
Can ve really rely on it always being `..` here? Isn't this dependent on where you place your `build` folder?

For example. I mean the usual is to have it under `project_root/build/` so then `../` would be `project_root/` ... But what if the `build` folder is completly outside of the `project_root` does this still work then?